easybuild/easyconfigs/c/cclib/cclib-1.6.3-foss-2020a-Python-3.8.2.eb

```
@@ -0,0 +1,39 @@
+# Authors:: Jack Perdue <j-perdue@tamu.edu> - TAMU HPRC - https://hprc.tamu.edu
+
+easyblock = 'PythonBundle'
+
+name = 'cclib'
+version = '1.6.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://cclib.github.io/'
+
+description = "parsers and algorithms for computational chemistry"
```

I suggest to not change the description of the easyconfig to keep it consistent
```suggestion
description = """cclib is a Python library that provides parsers for computational chemistry log files.
 It also provides a platform to implement algorithms in a package-independent manner.
"""
```